src/Data/Nat/Properties.agda

```diff
@@ -1558,6 +1562,9 @@ m≤n⇒n∸m≤n (s≤s m≤n) = m≤n⇒m≤1+n (m≤n⇒n∸m≤n m≤n)
   (m + n) ∸ o          ≡⟨ +-∸-assoc m o≤n ⟩
   m + (n ∸ o)          ∎
 
+[m-n-1]+1≡[m-n] : ∀ {m n} (n<m : n < m) → suc (m ∸ suc n) ≡ m ∸ n
```

Comment:
The name of this proof should match its type, i.e. `1+[m∸[1+n]]≡m∸n`. Also it's basically free to make it more general, i.e. make `1` to some natural `o` instead. 

However, looking at it, is it really worth adding it as a standalone item? The proof isn't exactly long?

Reply:
Re: name: good, will do
Re: generality: again, good, but I had been finding it a common case, so wanted to put it on its  own footing.

The proof isn't long, but I found that I kept using it in this formulation (rather than anything more general;l but crucially, IIRC, across multiple PRs associated with `Binomial`), and that the lemma it derives from has a *completely* unmemorbale name (to me at least). 

But I will clean it up, if necessary by having a `private` lemma elsewhere.

Reply:
Yes, I think it's avoidable here.